examples/nixnet/CAN_SIGNAL_SINGLE_POINT_OUTPUT.py

```diff
@@ -0,0 +1,120 @@
+r""" Write Signal Data.
+
+ This example writes a signal value when a keyboard character is pressed.
+ This is used to demonstrate a signal single point output session. 
+ This example uses hardcoded signal names that use the NIXNET_example database. 
+
+The gRPC API is built from the C API. NI-XNET documentation is installed with the driver at:
+  C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niXNET\Documentation\NIXNETCref.chm
+Getting Started:
+
+To run this example, install "NI-XNET Driver" on the server machine:
+  https://www.ni.com/en-in/support/downloads/drivers/download.ni-xnet.html
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC
+Client" wiki page:
+  https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Refer to the NI XNET gRPC Wiki for the latest C Function Reference:
+  https://github.com/ni/grpc-device/wiki/NI-XNET-C-Function-Reference
+
+ Running from command line:
+
+Server machine's IP address, port number, and resource name can be passed as separate command line
+arguments.
+  > python CAN_SIGNAL_SINGLE_POINT_OUTPUT.py <server_address> <port_number>
+If they are not passed in as command line arguments, then by default the server address will be
+"localhost:31763"
+"""
+
+import sys
+
+import getch
+import grpc
+import nixnet_pb2 as nixnet_types
+import nixnet_pb2_grpc as grpc_nixnet
+
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    SERVER_ADDRESS = sys.argv[1]
+if len(sys.argv) >= 3:
+    SERVER_PORT = sys.argv[2]
+
+# Parameters
+INTERFACE = "CAN1"
+DATABASE = "NIXNET_example"
+CLUSTER = "CAN_Cluster"
+SIGNAL_LIST = "CANEventSignal1,CANEventSignal2"
+NUM_SIGNALS = 2
+SUCCESS = 0
+
+# Display Error Function
+def display_error_andexit(status, source):
+    """Display error message and exit."""
+    response = client.StatusToString(nixnet_types.StatusToStringRequest(status_id=status))
+    print("\n\nERROR at {}!\n{}\n", source, response.status_description)
+    print("\nExecution stopped.\nPress any key to quit\n")
```

Comment:
Doesn't look like this line is going to wait here for user input. So this message doesn't make sense. Make these changes to keep consistency with other gRPC examples we ship out of this repo.

- I think instead of printing error and exiting, you can `raise Exception(response.status_description)` here. That will also take care of printing call stack, so users would know which line resulted in error.

- Also, put main body of this file in try-except-finally block. `except` can catch RPC errors and print RPC specific error message. `finally` block can be used for closing session if  `session_reference.id` is not zero (id != 0 makes sure that error wasn't during session initialization and we should close the sesssion).

- Rename this method to check_for_error and check for `status != SUCCESS` here to raise exception conditionally. Callers below can call this API unconditionally after each gRPC calls and print messages you have in success scenarios after the `check_for_error`. Those won't be executed in case of errors because of `raise Exception` here. 

Look at other examples to refer to this pattern.